Formula/gambit-scheme.rb

```diff
@@ -0,0 +1,127 @@
+class GambitScheme < Formula
+  desc "Gambit Scheme"
+  homepage "http://gambitscheme.org"
+  url "https://github.com/gambit/gambit.git", :tag => "v4.9.0"
+
+  depends_on "gcc@6"
```

Comment:
This needs to build with latest gcc (`depends_on "gcc"`)  to be considered for inclusion in Homebrew core.

Reply:
We need a comment stating why this cannot compile with `clang`. And then `fails_with :clang`.